pkg/config/config.go

```diff
@@ -1235,32 +1239,32 @@ func Reload() (*Config, error) {
 	return defConfig()
 }
 
-func (c *Config) ActiveDestination() (uri, identity string, err error) {
+func (c *Config) ActiveDestination() (uri, identity string, machine bool, err error) {
```

Comment:
Absolutely non-blocking: Wouldn’t it ultimately be easier to return a `Config.Destination`, even if manufactured locally?

We already have a struct for these three things, so we might as well use it so that we don’t need to worry about correctly naming items. Maybe a copy to guarantee the caller can’t modify the config.

Reply:
this does not make sense I do not think to do since we funnel into the `PodmanConfig` rather than a `config.Config`